includes/migration/wc/class-fs-wc-migration-endpoint.php

```diff
@@ -22,6 +23,27 @@ class FS_WC_Migration_Endpoint extends FS_Migration_Endpoint_Abstract {
 		 */
 		private static $_instance;
 
+		/**
+		 * @var stdClass Order data Example in /tests/order.txt
+		 */
+		private $_order;
+
+		/**
+		 * @param string $name Property to return
+		 * @return mixed|null Value if property exists else null
+		 */
+		public function __get( $name ) {
```

Comment:
@shramee why do you need external "magic" access? It's not a good practice in general.